src/driver/codemirror/index.ts

```diff
@@ -53,6 +53,10 @@ module.exports = {
                                 };
                                 await linkFolderOptionFunc(context, cm, val, old);
 
+                                if (settings.formattingBar) {
+                                    initFormattingBar(context, cm);
+                                }
+
```

Comment:
This was originally moved here because one of the other parts of the plugin was throwing an exception when enabled (causing `initFormattingBar` not to load).

I should determine whether this is still the case and, if so, modify the broken code to not load in CodeMirror 6.